src/rust/lib/graph-editor/src/component/visualization/foreign/java_script/definition.rs

```diff
@@ -93,36 +93,46 @@
 //!   area automatically. The `size` parameter contains two fields `width` and `height` expressed in
 //!   pixels.
 
-// FIXME: Make sure the above definition is implemented in the JS API, including the `Visualization`
-//        superclass, its `dom` field and the `setPreprocessor` method (connected to Rust side).
+// TODO: Connect the `setPreprocessor` method on Rust side.
```

Comment:
is the setPreprocessor method from JS not connected to the FRP yet? Can we please do it in this PR? This is the last piece to have fully functional specification connected. Moreover, after talking with engine today, it seems that we will need this functionality very, very soon.